### DISCUSSION TITLE
Grey area between window and status

### DISCUSSION THREAD

**Discussion by odiumuniverse** [catppuccin/tmux#259] [Help] [answered]
I use light theme everywhere and my background is always white(xFFFFFF), but I don't know how to configure this here and it's pissing me off, what command is used to configure this? It's really a very small colour difference, but it really pisses me off. I put the catpuccin latte theme on, but it doesn't match my neovim theme
<img width="1543" alt="Screenshot 2024-07-23 at 23 01 28" src="https://github.com/user-attachments/assets/5687d2e2-350e-4d74-b5f3-cad245fc0b0d">

**Comment by vdbe** [comment, marked as answer]
I think you mean the status bar color to change that see: https://github.com/catppuccin/tmux/blob/004467850b44983c7b033cf5e41c0db75d03742b/README.md#override-the-default-status-background-color

As for the the mismatch with neovim if you also use the catppuccin latte theme there see: https://github.com/catppuccin/nvim?tab=readme-ov-file#for-tmux-users
